waspc/src/Wasp/AppSpec/ExtImport.hs

```diff
@@ -21,7 +22,15 @@ data ExtImport = ExtImport
     -- | Path from which we are importing.
     path :: ExtImportPath
   }
-  deriving (Show, Eq, Data)
+  deriving (Show, Eq, Data, Generic)
+
+instance FromJSON ExtImport where
```

Comment:
Here I manually implemented JSON parsing in order to explain how to parse the StrongPath in question.

Reply:
You confused me a bit since you changed the values, but I get now that you went from
```json
{
  "name": {
    "contents": "myAction",
    "tag": "ExtImportModule"
  },
  "path": "folder/file.js"
}
```
to
```json
{
  "name": "myAction",
  
  "path": "folder/file.js"
}
```

Reply:
> You confused me a bit since you changed the values

Yeah, I fixed it now.